### ISSUE TITLE
How do we want to illustrate our algorithm?

### ISSUE THREAD

**Issue by koosvanamerongen** [MatthewPeterKelly/FluenceMapping#26]
In particular, we need to illustrate the relation between inner- and outerloops.
Flowchart, pseudo code, ...?

**Comment by MatthewPeterKelly** [comment]
This issue depends on #22 - we should decide the search space technique.

**Comment by koosvanamerongen** [comment]
Now we've decided on issue #22 (fixed max dose rate in main body, variable dose rate with CMAES in appendix): any ideas on this? I guess it suffices to describe the relation between inner leaf trajectory and outer dose rate loops without the need for any figure or pseudocode. Shall we summarize the inner loop leaf trajectory optimization somehow?

**Comment by MatthewPeterKelly** [comment]
I think that it would be good to have one paragraph in the introduction that explains how the nested optimization works, and then goes on to say that we will fix the dose rate as a simplifying assumption for the main body of the paper. It could then point to the appendix for more details, if we end up adding them there.

**Comment by koosvanamerongen** [comment]
Yes, that's a good thing to do for sure. I don't think we need to support that paragraph with a flowchart or some pseudocode of the full algorithm, do we?

**Comment by davidcraft** [comment]
I agree with koos, no need for a flow chart for the nesting. I put in a simple sentence that says we deal with the dose rate loop in the appendix, I think that should suffice.
